backend/sozisel/priv/repo/migrations/20210419115640_create_events.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Sozisel.Repo.Migrations.CreateEvents do
+  use Ecto.Migration
+
+  def change do
+    create table(:events) do
+      add :name, :string, null: false
+      add :start_minute, :integer
```

Comment:
Make everything that is required `null: false`, I think that start_minute is required but I may be wrong.